test/bxml-responses/hangup.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Response>
+     <Hangup/>
```

Comment:
does this need to be [`<hangup></hangup>`](http://ap.bandwidth.com/docs/xml/hangup/) ?